MonoGame.Framework/SDL/SDL2.cs

```diff
@@ -0,0 +1,717 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.Text;
+using System.Runtime.InteropServices;
+
+internal static class Sdl
+{
+    private const string NativeLibName = "SDL2.dll";
```

Comment:
If the dev is building for AnyCPU, how would the detection work? The game
could be run on either architecture.  This is where Mono actually has the
upper hand over Microsoft in that the dll.config file would handle this
runtime selection of library based on the user's architecture. Microsoft
don't support that.
On 15 Apr 2016 08:35, "Tom Spilman" <notifications@github.com> wrote:

In MonoGame.Framework/SDL/SDL2.cs
<https://github.com/mono/MonoGame/pull/4755#discussion_r59803592>:

> @@ -0,0 +1,717 @@
> +﻿// MonoGame - Copyright (C) The MonoGame Team
> +// This file is subject to the terms and conditions defined in
> +// file 'LICENSE.txt', which is part of this source code package.
> +
> +using System;
> +using System.Text;
> +using System.Runtime.InteropServices;
> +
> +internal static class Sdl
> +{
> +    private const string NativeLibName = "SDL2.dll";

True... we know that is coming soon.

—
You are receiving this because you were mentioned.

Reply to this email directly or view it on GitHub
<https://github.com/mono/MonoGame/pull/4755/files/b02e831910f420299c3838712a3aed80f51ed1c0#r59803592>